src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Models/VersionedEntityDeletionOrderUtilTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using GovUk.Education.ExploreEducationStatistics.Admin.Models;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Common.Services.CollectionUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Models
+{
+    public class VersionedEntityDeletionOrderUtilTests
+    {
+        [Fact]
+        public void Sort()
+        {
+            var entity1Version1Id = "entity-1-version-1";
+            var entity1Version2Id = "entity-1-version-2";
+            var entity1Version3Id = "entity-1-version-3";
+            var entity1Version4Id = "entity-1-version-4";
+            var entity1Version5Id = "entity-1-version-5";
+            var entity1Version6Id = "entity-1-version-6";
+            var entity1Version7Id = "entity-1-version-7";
+            var entity1Version8Id = "entity-1-version-8";
+            var entity2Version1Id = "entity-2-version-1";
+            var entity2Version2Id = "entity-2-version-2";
+            var entity2Version3Id = "entity-2-version-3";
+            var entity3Version1Id = "entity-3-version-1";
+            var entity3Version2Id = "entity-3-version-2";
+            var entity4Version3Id = "entity-4-version-3";
+            var entity4Version6Id = "entity-4-version-6";
+
+            var versions = new List<IdAndPreviousVersionIdPair<string>>
+            {
+                new(entity1Version6Id, entity1Version5Id),
+                new(entity3Version2Id, entity3Version1Id),
+                null,
+                new(entity1Version2Id, entity1Version1Id),
+                new(entity4Version3Id, "entity-4-version-2"),
```

Comment:
How come you're using string literals here?

Reply:
I wanted to convey that these entries didn't have any associated previous version in the list of returned entities.   More as an edge case rather than anything I'd expect to happen.

Reply:
Would you say that I should add a comment to explain it in the test?

Reply:
I'm ok with leaving it - does make sense now you say it.